app/models/foreman_docker/docker.rb

```diff
@@ -73,23 +69,19 @@ def provider_friendly_name
     end
 
     def create_container(args = {})
-      options = vm_instance_defaults.merge(args)
+      options = vm_instance_defaults.merge(args) { |_, default, new| new.empty? ? default : new }
```

Comment:
This was changed so that when no Cmd is specified, it uses the default. Otherwise a `.merge` will not replace a `[]` as it's not `nil`.